draft-sharabayko-mops-srt.md

```diff
@@ -511,12 +511,12 @@ Encryption Field: 16 bits.
 : Block cipher family and key size. The values of this field are
   described in {{handshake-encr-fld}}. The default value is AES-128.
 
- | Value | Cipher family and key size |
- | ----- | :--------------------------: |
- |     0 | No Encryption Advertised     |
- |     2 | AES-128                      |
- |     3 | AES-192                      |
- |     4 | AES-256                      |
+| Value | Cipher family and key size |
```

Comment:
@J-Rogmann Throughout the PR, there are changes not related to "Data Transmission Modes" section. What's interesting I do not see any essential changes. Please remove them appropriately and make a commit so that we have changes for "Data Transmission Modes" section.

Reply:
I did not made these changes and i assume, that maybe the typora editor was removing leading "spaces" at the beginning of certain lines